tap_postgres/sync_strategies/incremental.py

```diff
@@ -24,6 +26,15 @@ def fetch_max_replication_key(conn_config, replication_key, schema_name, table_n
             LOGGER.info("max replication key value: %s", max_key)
             return max_key
 
+
+def get_min_db_in_progress_transaction(conn_info):
+    with post_db.open_connection(conn_info) as conn:
+            with conn.cursor(cursor_factory=psycopg2.extras.DictCursor, name='stitch_cursor') as cur:
+                select_sql = """select txid_snapshot_xmin(txid_current_snapshot())"""
```

Comment:
Does this get the minimum tx id of all currently running queries, or only for the currently running queries in the session
https://www.postgresql.org/docs/9.5/functions-info.html#FUNCTIONS-TXID-SNAPSHOT

Some testing suggests it might be for the session only ... more time needed for testing

Reply:
what do you mean by "session"? 
can you also elaborate what testing have you done which makes you think that "session only"?

> Does this get the minimum tx id of all currently running queries
based on my testing, it applies for all currently running queries.

Reply:
Thank you @hnatiukdm for clearing that up ...